tests/crontab/Cli/ExecutableTest.php

```diff
@@ -19,40 +21,42 @@
  */
 class ExecutableTest extends TestCase
 {
+    protected const SF_CRONTAB_TEST_FILES = __DIR__ . '/../../files';
```

Comment:
To fix warning message, adding this const value to replace the `define` const on `tests/bootstrap.php` file:

```
FILE: /home/lee/crontab/tests/bootstrap.php
--------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
--------------------------------------------------------------------------------
 1 | WARNING | A file should declare new symbols (classes, functions,
   |         | constants, etc.) and cause no other side effects, or it should
   |         | execute logic with side effects, but should not do both. The
   |         | first symbol is defined on line 5 and the first side effect is
   |         | on line 3.
--------------------------------------------------------------------------------
```